### PR TITLE
Add SkyStarsRC F405 AIO

### PR DESCRIPTION
**Pull request by howels**
Add the new SkyStarsRC board.

### CODE REVIEW

File: configs/default/SKST-SKYSTARSF405_AIO.config [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+dma pin C09 0
+# pin C09: DMA2 Stream 7 Channel 7
+
+
+# master
+set baro_bustype = SPI
+set baro_spi_device = 2
+set adc_device = 2
+set blackbox_device = SPIFLASH
+set dshot_burst = AUTO
```
Comment (howels) [inline configs/default/SKST-SKYSTARSF405_AIO.config:110]: the AUTO lines might not be required @haslinghuis is that correct?

File: configs/default/SKST-SKYSTARSF405_AIO.config [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+set dshot_bitbang = AUTO
+set current_meter = ADC
```
Comment (howels) [inline configs/default/SKST-SKYSTARSF405_AIO.config:112]: likewise I think the defaults for current and voltage are already ADC

**Comment by howels** [comment]
@hongzhou-zeng please verify this new SkyStars board.

**Comment by howels** [comment]
Suspect there are some defaults included here - for example `dshot_burst`, `dshot_bitbang`, `current_meter` and `battery_meter` look like they may be redundant.

**Comment by howels** [comment]
Reduced the re-stating of defaults to remove these which are set to defaults for F405:

```
set dshot_bitbang = AUTO
set current_meter = ADC
set battery_meter = ADC
set ibata_scale = 400
set beeper_inversion = ON
set beeper_od = OFF
set system_hse_mhz = 8
```

File: configs/default/SKST-SKYSTARSF405AIO.config [hunk 3]
```diff
@@ -0,0 +1,117 @@
+# Betaflight / STM32F405 (S405) 4.3.2 Nov 28 2022 / 07:26:30 (60c9521) MSP API: 1.44 # config: manufacturer_id: SKST, board_name: SKYSTARSF405, version: a1066f52, date: 2022-11-15T14:13:48Z 
+
+board_name SKYSTARSF405AIO
+manufacturer_id SKST
+
+#define USE_ACCGYRO_BMI270
+#define USE_FLASH_W25Q128FV
+#define USE_MAX7456
+
+# name: -
```
Comment (haslinghuis) [inline configs/default/SKST-SKYSTARSF405AIO.config:10]: Please remove

**Review by haslinghuis — APPROVED** [review]

**Review by KarateBrot — APPROVED** [review]

**Pull request merged**